diabetes_regression/scoring/deployment_config_aks.yml

```diff
@@ -1,16 +0,0 @@
-computeType: AKS
-autoScaler:
```

Comment:
Why is this AKS config deleted ?

Reply:
Environment definition is moved to Python script.
AML Python SDK does not provide a way to define an environment in a YAML file, only in code (I have checked - the YAML parsing functionality is implemented directly in the AZ CLI ML extension - would be a good thing to propose to add in the SDK!)

Reply:
Please change this to CLI and config files instead of SDK to deploy model. We want to demonstrate the ease of CLI to deploy models

Reply:
We would still not have YML files anymore, and need an SDK script to generate the environment, and put the name in a json. Then we would be able to call the CLI. We would essentially remove a line from the existing script, but then have a delay while the extension gets installed every time for every build. Having a program generate input for a CLI would not seem to be a meaningful option to users.

Ideally the service itself should support such declarative environment versioning, rather than the imperative (additive) version behavior available today which is hard to use with DevOps.